src/commands/InitCommand.js

```diff
@@ -84,7 +84,7 @@ export default class InitCommand extends Command {
     const versionLocation = this.repository.versionLocation;
     if (FileSystemUtilities.existsSync(versionLocation)) {
       this.logger.info("Removing old VERSION file.");
-      FileSystemUtilities.unlinkSync(versionLocation, "0.0.0");
+      FileSystemUtilities.unlinkSync(versionLocation);
```

Comment:
Code [evolves](https://github.com/lerna/lerna/commit/1bc550438428afdb939ddf0a30907f891a3222e1). ;)

```diff
-        this.logger.info("Creating VERSION file.");
-        FileSystemUtilities.writeFileSync(versionLocation, "0.0.0");
...
+      this.logger.info("Removing old VERSION file.");
+      FileSystemUtilities.unlinkSync(versionLocation, "0.0.0");
```

This looks safe to remove.  Nice catch.